picard/script.py

```diff
@@ -479,7 +502,14 @@ def func_copymerge(parser, new, old):
         old = "~" + old[1:]
     newvals = parser.context.getall(new)
     oldvals = parser.context.getall(old)
-    parser.context[new] = newvals + list(set(oldvals) - set(newvals))
+    result = []
+    for x in newvals:
+        if x not in result:
+            result.append(x)
+    for x in oldvals:
+        if x not in result:
+            result.append(x)
+    parser.context[new] = result
```

Comment:
The goal is to get each element once in the final list, right ?

If you don't mind order:

```python
parser.context[new] = list(set(oldvals + newvals))
```

If you mind order, use https://github.com/metabrainz/picard/blob/master/picard/util/__init__.py#L266 `uniqify()`